packages/wonder-blocks-tooltip/components/tooltip.integration.test.js

```diff
@@ -0,0 +1,108 @@
+// @flow
+import * as React from "react";
+
+import {mount, unmountAll} from "../../../utils/testing/mount.js";
+import Tooltip from "./tooltip.js";
+import TooltipBubble from "./tooltip-bubble.js";
+import TooltipAnchor from "./tooltip-anchor.js";
+import {TooltipDisappearanceDelay} from "../util/constants.js";
+
+describe("tooltip integration tests", () => {
+    beforeEach(() => {
+        unmountAll();
+        jest.useFakeTimers();
+        // jest.clearAllMocks().resetModules();
+    });
+
+    it("should set timeoutId be null when TooltipBubble is active", () => {
+        // Arrange
+        const wrapper = mount(
+            <Tooltip content="hello, world">an anchor</Tooltip>,
+        );
+        const anchor = wrapper.find(TooltipAnchor).getDOMNode();
+
+        // Act
+        anchor && anchor.dispatchEvent(new FocusEvent("mouseenter"));
```

Comment:
Simulating the events is better than for integration tests than setting state since it exercises more parts of the system.  In this case we're verifying that the event handlers are behaving as we expect.